ui/src/app/course/components/editor/editor.component.html

```diff
@@ -1,44 +1,58 @@
 <div class="course-editor row">
 	<div class="col-12">{{titleEditor}}</div>
-	<form class="col-12" (submit)="onSubmit()">
+	<form class="col-12" (submit)="onSubmit()" [formGroup]="formGroup">
 		<app-loader *ngIf="isLoading"></app-loader>
 		<div class="row" *ngIf="!isLoading">
-			<div class="col-xs-12 col-sm-6">
+			<div class="col-12">
 				<div class="form-group">
 					<label class="" for="name">title</label>
-					<input class="form-control" [formControl]="course.name" type="text" id="name" name="name">
+					<input
+						class="form-control"
+						[formControlName]="formFields.name"
+						type="text"
+						id="name"
+						name="name">
 				</div>
 			</div>
-			<div class="col-xs-12 col-sm-6">
+			<div class="col-12">
 				<div class="form-group">
-					<label class="" for="date">date</label>
-					<input class="form-control" [formControl]="course.date" type="text" id="date" name="date">
+					<label class="" for="description">description</label>
+					<textarea
+						class="form-control form-text"
+						[formControlName]="formFields.description"
+						id="description"
+						name="description"></textarea>
 				</div>
 			</div>
-			<div class="col-12 col-sm-6">
+			<div class="col-12">
 				<div class="form-group">
-					<label class="" for="length">length</label>
-					<input class="form-control" [formControl]="course.length" type="text" id="length" name="length">
-				</div>
-			</div>
-			<div class="col-12 col-sm-6">
-				<div class="form-check">
-					<input class="form-check-input" [formControl]="course.isTopRated" type="checkbox" id="isTopRated"
-					       name="isTopRated">
-					<label class="form-check-label" for="isTopRated">
-						isTopRated
-					</label>
+					<label class="" for="date">date</label>
+					<input
+						class="form-control"
+						[formControlName]="formFields.date"
+						type="text"
+						id="date" name="date">
 				</div>
 			</div>
 			<div class="col-12">
 				<div class="form-group">
-					<label class="" for="description">description</label>
-					<textarea class="form-control form-text" [formControl]="course.description" type="text" id="description"
-					          name="description"></textarea>
+					<label class="" for="length">Duration</label>
+					<app-input-duration
+						name="length"
+						id="length"
+						[formControl]="formGroup.controls.duration"
```

Comment:
Duration have to show duration in HH:MM
![image](https://user-images.githubusercontent.com/81073/73395230-1e877700-42f0-11ea-8b76-ae302c0f7caa.png)

Reply:
done https://github.com/taraswww777/AngularGlobalMentoringProgram2019/commit/8477d9bc